README.md

```diff
@@ -26,7 +26,7 @@ The stack:
 - [route-snapper](https://github.com/dabreegster/route_snapper/) to draw routes
   snapped to existing roads
 
-You'll need
+To Run Locally You'll need:
```

Comment:
Oops, I just merged a quick PR this morning to move these instructions to `docs/developer.md`. Could you please rebase and update there?